src/components/page-list/page-list.jsx

```diff
@@ -100,6 +100,11 @@ export default class PageList extends React.Component {
       return;
     }
 
+    if (event.ctrlKey || event.metaKey) {
+      window.open(`/page/${page.uuid}`, '_blank');
+      return;
+    }
+
     this.props.history.push(`/page/${page.uuid}`);
```

Comment:
Might be good to share the same URL string between these two calls with a variable.